spec/amadeus/namespaces.test.js

```diff
@@ -74,8 +74,8 @@ describe('Namespaces', () => {
       expect(amadeus.referenceData.location('ALHR').get).toBeDefined();
       expect(amadeus.referenceData.locations.get).toBeDefined();
       expect(amadeus.referenceData.locations.airports.get).toBeDefined();
-      expect(amadeus.referenceData.locations.pointsOfInterest.get).toBeDefined();
-      expect(amadeus.referenceData.locations.pointsOfInterest.bySquare.get).toBeDefined();
+      expect(amadeus.referenceData.locations.pointsOfInterest('XXX').get).toBeDefined();
+      expect(amadeus.referenceData.locations.pointsOfInterest('XXX').bySquare.get).toBeDefined();
```

Comment:
Why do you pass a param for a global search bySquare?

Reply:
Fixed